src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs

```diff
@@ -68,7 +68,7 @@ namespace NuGetVSExtension
     public sealed class NuGetPackage : AsyncPackage, IVsPackageExtensionProvider, IVsPersistSolutionOpts
     {
         // It is displayed in the Help - About box of Visual Studio
-        public const string ProductVersion = "4.6.0";
+        public const string ProductVersion = "5.0.0";
```

Comment:
If you change this to 

    public static readonly string ProductVersion = typeof(NuGetPackage).Assembly.Version.ToString();

then we won't need to update it when the version number changes again.

Reply:
You can't do that. 

This is specified in an attribute, so the value has to be a constant.

Reply:
ok, in that case I'd like to see a comment in build/config.props that says when changing any of the NuGetVersion props, also edit src/NuGet.Clients/NuGet.Tools/NuGetPackage.cs.

We can remove the comment if we figure out how to automate it in the future.